### PR TITLE
fix(openapi): wallet and ecommerce-for-io. Add 401 to openapi exposed to the app

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Add 401 response code to openapi definitions for `ecommerc-for-io` and `payment-wallet` api groups

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl [hunk 1]
```diff
@@ -90,6 +90,16 @@
...
           },
+          "401": {
+            "description": "Unauthorized, access token missing or invalid",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
```
Comment (pietro-tota) [inline src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl:101]: are you sure we give a ProblemJson response body in case of 401?

**Review by infantesimone — DISMISSED** [review]
just comments related to response body

File: src/domains/pay-wallet-app/api/payment-wallet/v1/_openapi.json.tpl [hunk 2]
```diff
@@ -422,6 +478,16 @@
...
           },
+          "401": {
+            "description": "Unauthorized: the provided token is not valid or expired.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
```
Comment (pietro-tota) [inline src/domains/pay-wallet-app/api/payment-wallet/v1/_openapi.json.tpl:489]: remove also this body in order to be aligned with other api's

Comment (ciuffagianluca) [reply src/domains/pay-wallet-app/api/payment-wallet/v1/_openapi.json.tpl]: Fixed

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**